scripts/ci/kubernetes/docker/Dockerfile

```diff
@@ -33,12 +33,18 @@ RUN apt-get update -y && apt-get install -y \
         unzip \
     && apt-get clean
 
+
+RUN pip install --upgrade pip
+
 # Since we install vanilla Airflow, we also want to have support for Postgres and Kubernetes
 RUN pip install -U setuptools && \
     pip install kubernetes && \
     pip install cryptography && \
     pip install psycopg2-binary==2.7.4  # I had issues with older versions of psycopg2, just a warning
 
+COPY requirements.txt /tmp/requirements.txt
```

Comment:
I've removed these because we don't want to keep track of another `requirements.txt` file. When we do the `pip install /tmp/airflow.tar.gz` it will also install the dependencies.

Reply:
@Fokko So what this is doing is reducing the amount of time it takes when doing local development.

Basically right now every time we reload the docker image we have to pull in ALL of the airflow dependencies, which adds 1-3 minutes to each build. With this if the requirements haven't changed, then the lower levels of the docker container remain the same. This also reduces the amount of space these images would take in your docker registry (otherwise every single image has a copy of those dependencies)

Reply:
@dimberman Ah, okay. Makes sense for the development.

Regarding the image size, I'm pretty sure that the size will stay roughly the same. Right now all the dependencies are in the cached layer, otherwise they will be in the lower layer. Either way, the dependencies have to go in the image.
I would even argue that the image will be bigger in this case, since we are checking once more if all the dependencies are there at the `pip install` below.